infor-design/enterprise
# Soho 7968 pie fixes and app menu fixes

> Explain the **details** for making this change. What existing problem does the pull request solve?

Fixed issue on pie chart that it did not resize at the right time when App menu is toggled
Added mobile mode on pie chart to force Mobile pie charts for small widgets.
Found some components are missing an off so events multiply on every resize
Fixed an issue where app menu icon is pushed down with many items. #75
Added headerHamburger option to toggle menu for testing. SOHO-7968

> **Related issue (required)**:

https://jira.infor.com/browse/SOHO-7968
https://github.com/infor-design/enterprise-ng/issues/75

> **Steps necessary to review your pull request (required)**:

To Test this issue checkout 

http://localhost:4000/components/pie/test-legend-below.html
- this is the new mobile forced mode, shows legend below.

http://localhost:4000/components/pie/example-index?headerHamburger=true
- open menu
- ensure the pie resizes

http://localhost:4000/components/applicationmenu/test-filterable-many-items.html
- scroll down to see the icon is aligned

Any page...add ?headerHamburger=true to see the menu

> Other Details:

closes infor-design/enterprise-ng#75
